forms-flow-web/src/actions/actionConstants.js

```diff
@@ -125,7 +125,6 @@ const ACTION_CONSTANTS = {
   BPM_FORM_LIST_PAGE_CHANGE: "BPM_FORM_LIST_PAGE_CHANGE",
   BPM_FORM_LIST_LIMIT_CHANGE: "BPM_FORM_LIST_LIMIT_CHANGE",
   BPM_FORM_LIST_SORT_CHANGE: "BPM_FORM_LIST_SORT_CHANGE",
-  BPM_MAINTAIN_PAGINATION: "BPM_MAINTAIN_PAGINATION",
```

Comment:
Would Need to Maintain pagination from going from one page to another and coming back. Same applies for Search / filter . Reset at search/ filter / deletion. @Josephalexantony-aot / @AbijithS-aot / @sreehari-aot